src/components/CorporateCampaign/CampaignProgressBar.vue

```diff
@@ -86,6 +122,9 @@ export default {
 			}
 			return this.promoData?.promoFund?.displayName ?? null;
 		},
+		isMatchingCorporateCampaign() {
+			return this.isMatchingCampaign;
+		},
```

Comment:
@tiarascoleman my apologies for the confusion but this computed property is not needed.

In order to make your component inherit the existing `isMatchingCampaign` that we've established in CCLanding you'll need to do the following:
1. Update the ``  object [here](https://github.com/kiva/ui/blob/main/src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue#L793), by adding `isMatchingCampaign: this.isMatchingCampaign` along side the other key value pairs.
2. Update [CampaignLoanWrapper](https://github.com/kiva/ui/blob/main/src/components/Contentful/CampaignLoanWrapper.vue#L23) to pass a new prop into the CampaignProgressBar component by adding this: `:is-matching-campaign="componentProps.isMatchingCampaign"`
3. Update [CampaignProgressBar](https://github.com/kiva/ui/blob/main/src/components/CorporateCampaign/CampaignProgressBar.vue#L80) to add a new incoming prop declaration to match the prop your passing in:
```
		isMatchingCampaign: {
			type: Boolean,
			default: false
		},
```

With the listed additions in place you can remove this property b/c you will now have `isMatchingCampaign` available for use in your template and `this.isMatchingCampaign` available for use in your code.